Cheatsheets/index.md

```diff
@@ -11,6 +11,6 @@ Covers:
 
 ## [Converting Masterpages to Views](masterpagestoviews.md)
 
-Covers how you convert common syntax in Umbraco Masterpages to Umbraco Views.
+Covers how you can convert common syntax in Umbraco Masterpages to Umbraco Views.
```

Comment:
This implies that there are other ways to convert.  Could you add in the pull request which other ways are available?

Reply:
I agree there could be other ways. We can retain original style if you feel it makes sense